dspy/retrieve/databricks_rm.py

```diff
@@ -55,53 +47,72 @@ class DatabricksRM(dspy.Retrieve):
         )
 
         llm = dspy.OpenAI(model="gpt-3.5-turbo")
-        retriever_model = DatabricksRM(databricks_index_name = "your_index_name",
-        databricks_endpoint = "your_databricks_host_url", databricks_token = "your_databricks_token", columns= ["id", "field2", "field3", "text_vector"], k=3)
+        retriever_model = DatabricksRM(
+            databricks_index_name = "your_index_name",
+            docs_id_column_name="your_id_column",
+            text_column_name="your_text_column",
+            k=3
+        )
         dspy.settings.configure(lm=llm, rm=retriever_model)
         ```
 
-        Below is a code snippet that shows how to query the Databricks Direct Vector Access Index using the forward() function.
```

Comment:
The `forward()` function is not what users call to retrieve documents with DatabricksRM